l10n_it_vat_registries/static/description/index.html

```diff
@@ -475,10 +481,20 @@ <h2><a class="toc-backref" href="#toc-entry-6">Contributors</a></h2>
 <li>Alex Comba &lt;<a class="reference external" href="mailto:alex.comba&#64;agilebg.com">alex.comba&#64;agilebg.com</a>&gt;</li>
 </ul>
 </div>
+<div class="section" id="other-credits">
+<h2><a class="toc-backref" href="#toc-entry-7">Other credits</a></h2>
+<p>The development of this module has been financially supported by:</p>
+<ul class="simple">
+<li>Odoo Italia Network</li>
+<li>APS Odoo Italia</li>
+</ul>
+</div>
```

Comment:
Questo andrebbe nel commit in cui ha ripristinato il file `CREDITS`.

Reply:
e vabè